gspn.py

```diff
@@ -33,9 +32,13 @@ def __init__(self):
         self.__sparse_matrix_out = None
 
     def add_places(self, name, ntokens=None, set_initial_marking=True):
-        """
+        '''
+        Adds new places to the existing ones in the GSPN object. Replaces the ones with the same name.
 
-        """
+        :param name: (list str) denoting the name of the places
+        :param ntokens: (list int) denotes the maximum number of tokens of each place
+        :param set_initial_marking: (list int) denotes the initial marking of the petri net
```

Comment:
this is a bool, not a list, if True, the list ntokens will be set as the initial marking, and will not be set otherwise.